File: 1° Período - Introdução a Programação 1/Exercicios em C/Lista 1 - C/q10.c
//Faça um algoritmo que calcule a quantidade de litros necessários para uma certa viagem. 
#include<stdio.h>
int main(){
	int temp_gasto;
	float veloc_media, distancia, litros;
	printf("TEMPO GASTO: ");
	scanf("%d", &temp_gasto);
	printf("VELOCIDADE MEDIA: ");
	scanf("%f", &veloc_media);
	distancia=temp_gasto*veloc_media;
	litros=distancia/12;
	printf("Litros necessários: %f\n", litros);
}

File: 1° Período - Introdução a Programação 1/Exercicios em Python/Lista 1 - Python/q10.py
#Faça um algoritmo que calcule a quantidade de litros necessários para
#uma certa viagem. 

temp_gasto=int(input("Insira o tempo gasto na viajem em Horas: "))
veloc_media=int(input("Insira a velocidade média durante a viagem em Km/h: "))
distancia=temp_gasto*veloc_media
litros=round(distancia/12, 3)
print("Litros necessários: ", litros)
